basiclti/basiclti-tool/src/webapp/vm/lti_tool_site.vm

```diff
@@ -130,9 +130,9 @@ ${includeLatestJQuery}
             </select>
         </div>
 		#end
-		<table id="tools_site" class="tablesorter" cellspacing="0" summary ="$tlang.getString("content.table")">
+		<table id="tools_site" class="tablesorter tablesorter-blue hasFilters" cellspacing="0" summary ="$tlang.getString("content.table")">
```

Comment:
my only argument here is that the blue is a bit of a different design than everywhere else.... would rather have this neutral colors

Reply:
Agreed. "Blue" was already added as the theme in this file and for the lti_tool_system.vm table. I think we'd want to switch both to a new theme if we change it here.
`<script type="text/javascript">includeWebjarLibrary('jquery.tablesorter/2.27.7/dist/css/theme.blue.min.css');</script>`

Reply:
I say we merge this - so as to undo the regression - (i.e. restore it to how it was when it worked) - and then clean up colors later.

Reply:
New JIRA will be opened to address neutral theme of the table, thanks @profmikegreene